hollow/src/main/java/com/netflix/hollow/api/producer/HollowProducer.java

```diff
@@ -982,6 +999,17 @@ public Builder withListeners(HollowProducerListener... listeners) {
             return this;
         }
         
+        public Builder withValidationListener(HollowValidationListener listener) {
```

Comment:
Just making it easy to register a single listener at a time instead of collecting all into one call. This pattern is similar to the HollowProducer listener method in builder.

Reply:
yeah, existing pattern no so good.

When there is an method that takes "..." params, it already supports 0 or more so adding specific method to take 1 does not seem necessary.    Looks like we need to cleanup all this extra methods once we release Hollow 3.

Reply:
@davesu your point is reverse of @kineshsatiya. I agree with only providing multiple for the new API that I added. I will make that change.

Reply:
Done.